011-php-code-formatting.md

```diff
@@ -0,0 +1,94 @@
+# PHP Code Formatting
+
+We should implement the use of both the [Prettier PHP Plugin](https://github.com/prettier/plugin-php) for prettifying the look of our PHP code and the [PHP Coding Standards Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) tool for ensuring the resulting PHP code is written well, following specified code quality rules.
+
+**Discussion:** <https://github.com/DoSomething/rfcs/pull/11>
+
+## Problem
+
+For most of our repositories that include PHP we currently depend on [StyleCI](https://styleci.io) for formatting our code.
+
+While it has served our needs for the most part, this tool is not as convenient as some other tools available, and at times has even been a bit confusing to use.
+
+The setup for StyleCI includes adding a GitHub repository to the list of projects it should "check" automatically for each "push" to a pull request branch. The settings can be customized on the StyleCI web interface but each project can also contain their own configuration in a `.styleci.yml` file. Having multiple locations for configuration settings can be a bit confusing; not always being apparent if a project is checked by StyleCI if the local config file is not present.
```

Comment:
I don't know if it's necessarily worth explicitly pointing out our [various](https://dosomething.slack.com/archives/C3ASB4204/p1572615964090600) [kerfuffles](https://dosomething.slack.com/archives/CUQMU4Q6B/p1596116997002700?thread_ts=1596116997.002700) with unexpected default config changes which intersected with the configuration inconsistency we're pointing out here?

Reply:
Ah good point @mendelB ! I added more info mentioning lack of version control and linked to those Slack instances! <https://github.com/DoSomething/rfcs/pull/11/files#diff-f0c9652f43e84660cd5f067a9f73c7ddR53>